media/css/mozorg/home/home-new.scss

```diff
@@ -8,6 +8,27 @@ $image-path: '/media/protocol/img';
 
 @import '~@mozilla-protocol/core/protocol/css/includes/lib';
 
+// July 2024 Mozilla Foundation fundraiser nav CTA
+html.is-firefox {
+    .c-fx-users {
+        display: block;
+    }
+
+    .c-non-fx-users {
+        display: none;
+    }
+}
+
+html:not(.is-firefox) {
```

Comment:
**issue:** the `:not()` pseudo class is not supported in some older browsers such as IE, so could this result in 2 buttons showing in some browsers?

Reply:
Ah! Thanks for the good callout, I missed checking this one.
I went ahead and created a class in `site.js` called `.is-not-firefox` to be used in conjunction with the `.is-firefox` class name for when we need to be specific going forward. Let me know if this works better

Reply:
**Edit:** oops, tests are screaming. I think I need to revert that action and find another way lol

Reply:
Would it be simpler just to show the non-Firefox button by default and then swap out the other button for Firefox users? You might not need another selector that way?

Reply:
That's what I did :)